src/components/Details/__tests__/DetailObject.test.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { Icon } from 'patternfly-react';
+import { default as DetailObject } from '../DetailObject';
+import { DestinationWeight } from '../../../types/ServiceInfo';
+
+describe('DetailObject test', () => {
+  const detail: DestinationWeight = {
+    destination: {
+      host: 'reviews',
+      subset: 'v1',
+      port: {
+        number: 22,
+        name: 'ssh'
+      }
+    },
+    weight: 85
+  };
+
+  const mockRandom = () => {
+    const mockMath = Object.create(global.Math);
```

Comment:
Why is it necessary to mock the random ?
That means that there is some logic inside the code invoking the random() function ?

Reply:
Yep, it looks like DetailObject is using a random method:
https://github.com/xeviknal/kiali-ui/blob/4fb935a6d1002e74272efb2a737c6dadfaec65f0/src/components/Details/DetailObject.tsx#L21

Without that mocking test will fail every build.

Reply:
Oh, true, I didn't remember this logic, yes, it makes sense :-).